docs/ref/contrib/gis/gdal.txt

```diff
@@ -1873,7 +1873,7 @@ For commercial storage providers (e.g. ``/vsis3/``) the system should be
 previously configured for authentication and possibly other settings (see the
 `GDAL Virtual Filesystems documentation`_ for available options).
 
-.. _`GDAL Virtual Filesystems documentation`: https://gdal.org/user/virtual_file_systems.html
```

Comment:
The current links (with `.html`) work so I see no reason to change them. Moreover `gdal.org` uses them. Please revert changes in `docs`.

Reply:
I would also add `.html` suffixes to links updated in the source code.

Reply:
Got it, I've applied these changes